deno.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@denosaurs/pngs",
+  "version": "0.2.0",
+  "exports": "./mod.ts",
+  "tasks": {
+    "build": "deno run -A scripts/build.ts",
+    "fmt": "deno run -A scripts/fmt.ts",
+    "clean": "deno run -A scripts/clean.ts",
+    "lint": "deno run -A scripts/lint.ts"
+  },
+  "lint": {
+    "exclude": [
+      "pkg"
+    ]
+  },
+  "publish": {
+    "include": [
+      "mod.ts",
+      "wasm.js",
+      "README.md",
+      "LICENSE"
```

Comment:
Do we wan't to include non-code in the published version?

Reply:
doesn't jsr need those?

Reply:
I mean it shouldn't need to bundle those?